### DISCUSSION TITLE
Help figuring out which model to use

### DISCUSSION THREAD

**Discussion by guilhermetb** [sebanc/brunch#2147] [Q&A]
Which bin board from the website https://cros.tech/ should I use for the notebook:
Dell Inspiron N4050
Core i3 2310M 2100Mhz Sandy Bridge
4gb DDR3-1333 DDR3
Intel HD Grafics 3000
SSD 120GB
Drive CDROM
Bios version A08
Serial number 3LJDSR1

I've already tried the versions:
samus 83 89 91 - No video
asuka - The video appears but it is buggy,
rammus - It gets frozen on the GoogleOS logo

In asuka and rammus ctrl + alt + f2 enter in localhost login:, i type root and have prompt brunch~#

I am using MBR support

**Comment by radhekrishna-143** [comment]
i have the same configuration but unable to even install brunch can u help me which version I should go with
